.github/workflows/end-to-end-tests.yml

```diff
@@ -0,0 +1,36 @@
+name: "end-to-end_tests"
+on:
+    push:
+        branches: [develop]
+    pull_request:
+        branches: [develop]
+jobs:
+    "end-to-end_tests":
+        runs-on: ubuntu-latest
+        defaults:
+            run:
+                shell: bash
+                working-directory: ${{ github.workspace }}
```

Comment:
Hmm, this is maybe also the default, but I couldn't find good docs... Maybe not needed?